.github/workflows/diff_report.yml

```diff
@@ -180,8 +185,11 @@ jobs:
         DIFF="./contribution/checkstyle-tester/reports/diff"
         LINK="https://${{env.AWS_BUCKET_NAME}}.s3.${{env.AWS_REGION}}.amazonaws.com"
         aws s3 cp $DIFF s3://${{env.AWS_BUCKET_NAME}}/$FOLDER/reports/diff/ --recursive
-        echo $LINK/$FOLDER/reports/diff/index.html > message
-        
+        if [ ! -z "${{needs.parse_body.outputs.report_label}}" ]; then
+          echo "${{needs.parse_body.outputs.report_label}}: " > message
+        fi
+        echo $LINK/$FOLDER/reports/diff/index.html >> message
```

Comment:
Could `$LINK` and `$FOLDER` be double quoted, ex: `"$LINK"` and `"$FOLDER"`?

Reply:
same as in comment below

Reply:
Resolved.